modules/sdk-coin-islm/src/lib/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const validDenoms = ['aISLM'];
```

Comment:
There must be other denoms as well, like ISLMT as seen here -
https://testnet.ping.pub/haqq/tx/87C5D925D599D1E487523E1295566475BCDE7F94846A50B32413B4580497C6A3
Can you add all denoms. 
nit: should be in lowercase.

Reply:
Oh, ISLMT is just how the explorer shows it, the denom in tx json still is aISLM. The denom is expected in this format on chain so lowercase wouldn't have worked.